packages/material-ui/src/DialogContentText/DialogContentText.spec.tsx

```diff
@@ -16,13 +16,9 @@ const DialogContentTextTest = () => {
       <DialogContentText align="inherit" color="textPrimary" />
       <DialogContentText align="inherit" color="textSecondary" />
       <DialogContentText align="inherit" color="error" />
-      {/* @ts-expect-error */}
```

Comment:
Seems like the system props are a bit too relaxed and cannot catch these errors :\